Lib/unittest/test/testmock/testmagicmethods.py

```diff
@@ -429,7 +428,6 @@ def _dir(self):
             self.assertEqual(dir(mock), ['foo'])
 
 
-    @unittest.skipIf('PyPy' in sys.version, "This fails differently on pypy")
```

Comment:
If I understand correctly pypy vendors the modules from standard library and since this test passes now it's okay to remove and can be revisited if pypy devs want to have this : https://bitbucket.org/pypy/pypy/src/fd7960fb96180a26cbbc58a24b6e9a4d577d2e64/lib-python/3/unittest/test/testmock/testmagicmethods.py#lines-389

Reply:
Yep, this passes on pypy36 in the backport, so I can see no reason to skip this anymore.